test/code/scxcorelib/util/productdependencies.cpp

```diff
@@ -56,6 +56,11 @@ namespace SCXCoreLib
 
 namespace SCXSystemLib
 {
+    std::wstring OSTypeInfo_GetConfigPath()
+    {
+        return L"/etc/opt/microsoft/scx/conf/scxconfig.conf";
+    }
+
```

Comment:
With the change I mentioned above, then you can move this function down into the SCXProductDependencies namespace